code/parachain/node/src/command.rs

```diff
@@ -41,7 +41,7 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
 		// Must define the default chain here because `export-genesis-state` command
 		// does not support `--chain` and `--parachain-id` arguments simultaneously.
 		#[cfg(feature = "dali")]
-		"dali-dev" => Box::new(chain_spec::dali_dev()),
+		id if id.contains("dali") => Box::new(chain_spec::dali_dev(id)),
```

Comment:
not sure, what if we will have dali-rococo like others do tuned for rococo remote by parity?

Reply:
that is one of the reason why we use nix - we share and compose.

Reply:
then we will update the code for this case.